src/modules/authorization/sagas.ts

```diff
@@ -30,147 +30,163 @@ import {
 } from './actions'
 import { Authorization, AuthorizationAction, AuthorizationType } from './types'
 
-export function* authorizationSaga() {
-  yield takeEvery(
-    FETCH_AUTHORIZATIONS_REQUEST,
-    handleFetchAuthorizationsRequest
-  )
-  yield takeEvery(GRANT_TOKEN_REQUEST, handleGrantTokenRequest)
-  yield takeEvery(REVOKE_TOKEN_REQUEST, handleRevokeTokenRequest)
+export type AuthorizationSagaOptions = {
```

Comment:
To be consistent, what do you think about moving this to /types as we have on wallet?

Reply:
aah i didn't notice that on the wallet module, I made it like this by looking at the `translation` module 🦆 I'll move both to `/types` to keep it consistent.

Reply:
🏌️‍♂️

Reply:
done ✅